### PR TITLE
bring login_remaining in the admin

### PR DESCRIPTION
**Pull request by julsemaan**
# Description

Adds login remaining setting in user creation (single + multiple)
Adds display of remaining logins in the local accounts (no modification)
# Impacts

User creation + display
# Delete branch after merge

YES
# NEWS file entries
## New Features
- Added ability to limit the amount of logins when creating a local account
## Enhancements
- Added display of login remaining for local accounts in the user view

### CODE REVIEW

**Comment by julsemaan** [comment]
Please review

File: html/pfappserver/root/user/create.tt [hunk 1]
```diff
@@ -41,6 +42,7 @@
         <div class="alert alert-info">[% l('The usernames are constructed from the <b>prefix</b> and the <b>quantity</b>. For example, setting the prefix to <i>guest</i> and the quantity to <i>3</i> creates usernames <i>guest1</i>,<i> guest2</i> and <i>guest3</i>. Random passwords will be created.') %]</div>
         [% form_multiple.field('prefix').render | none %]
         [% form_multiple.field('quantity').render | none %]
+        [% form_single.field('login_remaining').render | none %]
```
Comment (jrouzierinverse) [inline html/pfappserver/root/user/create.tt]: form_multiple or form_single?

**Comment by jrouzierinverse** [comment]
Looks good except for that minor template issue.

**Comment by julsemaan** [comment]
fixed tempalte issue

**Pull request merged**